build_modules/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.1.4
```

Comment:
nit: technically these are new features (plus a bug fix) so we should increment the minor version number